src/layman/layer/geoserver/tasks.py

```diff
@@ -47,6 +48,7 @@ def refresh_wms(
     assert title is not None
     geoserver_workspace = wms.get_geoserver_workspace(workspace)
     geoserver.ensure_workspace(workspace)
+    metadata_url = micka_util.get_metadata_url(info['uuid'])
 
```

Comment:
- Remove CSW_RECORD_URL env and Python variables
- In layman_settings.py assert that `CSW_PROXY_URL` ends with `/csw`
- Get record URL from `CSW_PROXY_URL` by removing last `csw` from path and adding appropriate rest of the path
- `get_metadata_url` will be probably parametrized, returning either `basic` or `xml` in record path